src/dissect/dissect.c

```diff
@@ -757,6 +819,8 @@ static int run(int argc, char *argv[]) {
         _cleanup_(loop_device_unrefp) LoopDevice *d = NULL;
         int r;
 
+        setlocale(LC_ALL, "");
```

Comment:
why is this added? a comment above would be helpful

Reply:
it's actually a left-over: i wanted to force systemd-dissect into ASCII mode instead of unicode output mode. The way to do that is setting `LC_CTYPES=de_DE.ISO-8859-1`. But that's only honoured if `setlocale()` is called. 

But as it turns out that requires locales to actually be installed in the testing env, and I figure we shouldn't do that, as all the images would then grow huge. So I went or unicode-based testing, with the unicode check mark. Which I guess is actually better, since it allows us to test the unicode magic by side effect.

Anyway, will drop this here now. In the long run we should probably use setlocale() in all cmdline tools (i.e. not in daemons), so that turning off unicode output works.